modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -922,6 +922,9 @@ class NDB_Form_User_Accounts extends NDB_Form
      */
     private function _getEmailError($DB, $email)
     {
+        // remove illegal characters
+        $email = filter_var($email, FILTER_SANITIZE_EMAIL);
```

Comment:
The email stays the same and it solves this issue. Are there any reasons why this shouldn't be used?